examples/meam/msmeam/README.md

```diff
@@ -0,0 +1,9 @@
+To run Baske's test, do
```

Comment:
We don't want the examples folder "littered" with test inputs like this. There are some exceptions when the author of the package has written elaborate test/validation scripts. But in general, the better place for running consistency tests would be the unittest tree. See for example the unittest/force-styles/test_pair_list.cpp file.

Reply:
Unit test file is added